tf_fastmri_data/tests/datasets/non_cartesian_test.py

```diff
@@ -5,12 +5,12 @@
 
 
 
-@pytest.mark.parametrize('acq_type', ['radial', 'cartesian', 'spiral'])
-@pytest.mark.parametrize('dcomp', [True, False])
+@pytest.mark.parametrize('acq_type', ['radial'])
+@pytest.mark.parametrize('dcomp', [True])
 @pytest.mark.parametrize('multicoil', [True, False])
-@pytest.mark.parametrize('slice_random', [True, False])
+@pytest.mark.parametrize('slice_random', [False, True])
```

Comment:
```suggestion
@pytest.mark.parametrize('slice_random', [True, False])
```

Unnecessary change I think, let's keep it as is